managed-databases/redis/api-cli/managing-username-and-password.mdx

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  title: Managing the default user's username and password
+  description: Learn more about the default user's username and password for Managed Database for Redis™
+content:
+  h1: Managing the default user's username and password
+  paragraph: Learn more about the default user's username and password for Managed Database for Redis™
+tags: databases user redis username password
+categories:
+  - managed-databases
+  - redis
+---
+
+When you create a Redis™ Database Instance, a single, default database user is a created with [set permissions](/managed-databases/redis/reference-content/default-user-permissions).
```

Comment:
```suggestion
When you create a Redis™ Database Instance, you have to specify a user. It will be created with [set permissions](/managed-databases/redis/reference-content/default-user-permissions).
```